src/components/updatemenus/draw.js

```diff
@@ -181,8 +191,8 @@ function drawHeader(gd, gHeader, gButton, menuOpts) {
         .text('▼');
```

Comment:
Thought about that. I had mixed feelings. I'm not opposed to it. For comparison, system select menus look like this:

<img width="229" alt="2016-09-23 11 53 55" src="https://cloud.githubusercontent.com/assets/572717/18792413/6daba5da-8184-11e6-82bd-a03a1921c957.png">